less/variables.less

```diff
@@ -53,7 +53,7 @@
 @font-size-large:         (@font-size-base * 1.25);  // ~22px
 @font-size-small:         (@font-size-base * 0.836364);  // ~14.72px
 
-@font-size-h1:            (@font-size-base * 2.6);   // ~46px
+@font-size-h1:            (@font-size-base * 2.9);   // ~51px
 @font-size-h2:            (@font-size-base * 2.15);  // ~38px
```

Comment:
Is it possible to make H2-H4 smaller, and more differentiated, rather than increasing the size of the H1? These fonts are all rather huge. With a longish title, the H2s extend to two or sometimes three lines, like here: https://docs.puppet.com/pe/latest/release_notes_known_issues_console.html. It's also difficult to distinguish between H2 and H3, like here: https://docs.puppet.com/pe/latest/install_agents.html. You have to really be paying attention to font size to recognize that the H3s are children. Because we don't use indenting to indicate document structure, it seems important that the heading levels are very clearly distinguishable.